soh/src/code/random_experiment.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include <stdbool.h>
+
+//TODO: implement equivalent CSPRNG on Linux and Unix platforms
+#if defined(_WIN32)
+bool cryptoRandomInit(void);
+void cryptoRandomTearDown(void);
+unsigned long cryptoRandomUint32(unsigned long);
+#endif // _WIN32
+
+extern bool CSPRNG_INITIALIZED;
```

Comment:
I missed this before but the name should use or format.
So `gCspringInitialized`

Reply:
No problem. Got that fixed as well.

Also, yes, this change has been built and run successfully on Ubuntu 22.04 (Though as intended it does not appear in the menu and simply reverts to the default RNG on Linux as of now, as this is a Windows-only improvement for the time being).